examples/consumer/retry/order/main.go

```diff
@@ -41,6 +41,7 @@ func main() {
 		consumer.WithConsumeFromWhere(consumer.ConsumeFromFirstOffset),
 		consumer.WithConsumerOrder(true),
 		consumer.WithMaxReconsumeTimes(5),
+		consumer.WithStrategy(consumer.AllocateByAveragely),
```

Comment:
please modify consumer/simple.go for this

Reply:
have already added examples/consumer/strategy file